crt_portal/cts_forms/model_variables.py

```diff
@@ -82,6 +83,7 @@
     'commercial_or_public': [
         _('A physical or online location that does not provide disability accommodations'),
         _('Denied service or entry because of a perceived personal characteristic like race, sex, or religion'),
+        _('Denied an accommodation for a disability, including not being allowed to have a service animal <strong>in a commercial or public location</strong>'),
```

Comment:
Looks like the `<strong>` tags here are being rendered as text instead of mark-up on the page. I see you used `|safe` below, there may be an interaction between `gettext_lazy` and the templatetag.


<img width="500" alt="Screen Shot 2021-03-19 at 1 53 52 PM" src="https://user-images.githubusercontent.com/3485564/111822881-acd70780-88ba-11eb-9049-a223dedb46c9.png">

Reply:
Ahhh. I missed that these are rendered on the landing page as well 🤦 

Thanks for the catch!